doc/isis/frr-isis-sonic-yang-model-hld.md

```diff
@@ -0,0 +1,652 @@
+                }
+
+                leaf spf_short_delay {
+                    type uint16 {
+                        range "0..60000";
+                    }
+                    units "msec";
+                    must "../spf_init_delay and ../spf_long_delay and ../spf_hold_down and ../spf_time_to_learn or not(../spf_short_delay)" {
+                        error-message
+                            "SPF short delay must only be specified if all other SPF parameters are specified";
+                    }
+                    description
+                        "Delay used during SHORT_WAIT state";
+                }
+
+                leaf spf_long_delay {
+                    type uint16 {
+                        range "0..60000";
+                    }
+                    units "msec";
+                    must "../spf_init_delay and ../spf_short_delay and ../spf_hold_down and ../spf_time_to_learn or not(../spf_long_delay)" {
+                        error-message
+                            "SPF long delay must only be specified if all other SPF parameters are specified";
+                    }
+                    description
+                        "Delay used during LONG_WAIT state";
+                }
+
+                leaf spf_hold_down {
+                    type uint16 {
+                        range "0..60000";
+                    }
+                    units "msec";
+                    must "../spf_init_delay and ../spf_short_delay and ../spf_long_delay and ../spf_time_to_learn or not(../spf_hold_down)" {
+                        error-message
+                            "SPF hold down must only be specified if all other SPF parameters are specified";
+                    }
+                    description
+                        "Period of time without IGP events before considering IGP stable";
+                }
+
+                leaf spf_time_to_learn {
+                    type uint16 {
+                        range "0..60000";
+                    }
+                    units "msec";
+                    must "../spf_init_delay and ../spf_short_delay and ../spf_long_delay and ../spf_hold_down or not(../spf_time_to_learn)" {
+                        error-message
+                            "SPF time_to_learn must only be specified if all other SPF parameters are specified";
+                    }
+                    description
+                        "Maximum time needed to learn all of the events related to a failure";
+                }
+
+                leaf log_adjacency_changes {
+                    type boolean;
+                    default "false";
+                    description
+                        "Log changes to this instance's IS-IS adjacencies.";
+                }
+
+            } // list ISIS_GLOBAL_LIST
+
+        } // container ISIS_GLOBAL
+```
+
+##### SONiC ISIS Level
+ISIS Level Yang container is sonic-isis.yang.
+
+```
+        container ISIS_LEVEL {
```

Comment:
Suggest to use the name ISIS_LEVELS

Reply:
Updated the container name to ISIS_LEVELS